kapitan/cli.py

```diff
@@ -72,6 +72,10 @@ def main():
                                 default=from_dot_kapitan('compile', 'search-paths', ['.', 'lib']),
                                 metavar='JPATH',
                                 help='set search paths, default is ["."]')
+    compile_parser.add_argument('--filter-paths', '-J2F', type=str, nargs='+',
```

Comment:
Can we rename the flag to `--jinja2-filter-paths` since these are jinja2 specific?